app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivity.java

```diff
@@ -76,6 +90,29 @@ public void onClickFrom(View v){
         openSearchPage();
     }
 
+    public void onClickTransit(View v) {
+        mViewModel.setTransportType(ClassConstants.TRANSIT);
+        getRouteOptions();
+    }
+
+    public void onClickCar(View v) {
+        mViewModel.setTransportType(ClassConstants.DRIVING);
+        getRouteOptions();
+    }
+
+    public void onClickDisability(View v) {
```

Comment:
empty method?

Reply:
yeah we haven't implemented this yet, that'll be implemented in epic 5. I just put it so ppl know they'll have to write the logic here

Reply:
can you add a TODO with an issue number?

Reply:
yes issue #180